app/services/whatsapp/providers/whatsapp_cloud_service.rb

```diff
@@ -40,8 +45,25 @@ def media_url(media_id)
     "#{api_base_path}/v13.0/#{media_id}"
   end
 
+  def fetch_next_templates(response)
+    loop do
+      break unless response['paging'] && response['paging']['next']
+
+      response = HTTParty.get(response['paging']['next'])
+      push_templates(response)
+    end
```

Comment:
```suggestion
    while response['paging'] && response['paging']['next']
      response = HTTParty.get(response['paging']['next'])
      push_templates(response)
    end
```

Just one minor suggestion, everything else looks great

Reply:
I added `while` before, but Rubocop gives an issue.